### ISSUE TITLE
Nativescript app crashes on iOS version 13.4 while open a file with "myNSapp" It works well on version 13.3

### ISSUE THREAD

**Issue by SaifulZiko** [NativeScript/NativeScript#8579]
Hello , 
Since last 2 days my NS app crashes whenever I try to open or import a file with "myNSapp". It was working well but since my device updated to iOS version 13.4 , it crashes. Does anyone have any suggestions what can I do? 
It will be very helpful. 
Thanks in advance.
**error msg i got :** 
**"-[NSError init] called; this results in an invalid NSError instance. It will raise an exception in a future release. Please call errorWithDomain:code:userInfo: or initWithDomain:code:userInfo:. This message shown only once. 
CONSOLE LOG file:///app/app.js:305:0: iOS Import Content Null Error 
HTHangEventCreate: HangTracing is disabled. Not creating a new event."** 
package.json: 
  "tns-ios": {
      "version": "6.5.0"
    }
  },
  "dependencies": {
    "email-validator": "^2.0.4",
    "nativescript-bitmap-factory": "^1.7.1",
    "nativescript-camera": "^4.2.0",
    "nativescript-drawingpad": "^3.0.4",
    "nativescript-email": "^1.5.3",
    "nativescript-feedback": "^1.3.6",
    "nativescript-imagepicker": "^6.1.0",
    "nativescript-iqkeyboardmanager": "^1.4.0",
    "nativescript-local-notifications": "^3.1.2",
    "nativescript-masked-text-field": "^4.0.1",
    "nativescript-permissions": "^1.3.9",
    "nativescript-share-file": "^1.0.6",
    "nativescript-sqlite": "^2.3.3",
    "nativescript-theme-core": "^1.0.4",
    "nativescript-toasty": "^1.2.0",
    "nativescript-ui-listview": "^7.0.4",
    "nativescript-ui-sidedrawer": "^7.0.4",
    "nativescript-xml2js": "^0.5.2",
    "tns-core-modules": "^6.5.1",
    "tns-i18n": "0.0.10"
  },
  "devDependencies": {
    "babel-traverse": "^6.26.0",
    "babel-types": "^6.26.0",
    "babylon": "6.4.5",
    "lazy": "1.0.11",
    "nativescript-dev-webpack": "^1.2.1"
  }
}
**App Delegate :**
if ( application.ios ) {
   var DOCDelegate = (function (_super) {
        __extends(DOCDelegate, _super);
        function DOCDelegate() {
            _super.apply(this, arguments);
        }
           DOCDelegate.prototype.applicationDidFinishLaunchingWithOptions = function (application, launchOptions) {
            console.log("applicationWillFinishLaunchingWithOptions: " + launchOptions);
            return true;
        };
        DOCDelegate.prototype.applicationDidBecomeActive = function (application) {
            console.log("applicationDidBecomeActive: " + application);
        };
        DOCDelegate.prototype.applicationOpenURLOptions = function (application, url, options) {
            console.log("applicationOpenURLOptions: " + url);
            if ( applicationSettings.getBoolean("isAccessedUser", false) ){
                if ( url.pathExtension === "mcdoc" ) {
                    try {
                        var content = NSString.alloc().initWithContentsOfFile(url);
                        var errorIOS = NSError.new();
                       var content = NSString.alloc().initWithContentsOfURLEncodingError(url, NSUTF8StringEncoding, errorIOS);
                        if ( content !== null ) {
                            var json = getJsonFromContent( content );
                            if ( json ) {
                                var clName = json.Checklist.$.name;
                                importChecklist( clName, json );
                            }
                        }
                        else {
                            console.log("iOS Import Content Null Error");
                            dialogs.alert({
                                title: "Import Fehler",
                                message: "Beim Lesen der Datei ist ein Fehler aufgetreten.",
                                okButtonText: "OK"
                            });
                        }
                    } catch (error) {
                        console.log("iOS Import Error: " + error);
                        dialogs.alert({
                            title: "Import Fehler",
                            message: "Beim Lesen der Datei ist ein Fehler aufgetreten.",
                            okButtonText: "OK"
                        });
                    }
                }
                else {
                    setTimeout( function() {
                        dialogs.alert({
                            title: "Import Fehler",
                            message: "Das ist keine gültige  DOC App Datei.",
                            okButtonText: "OK"
                        });
                    }, 500);
                }
            }
            else {
                setTimeout( function() {
                    dialogs.alert({
                        title: "Import Fehler",
                        message: "Um eine Checkliste importieren zu können, benötigen Sie einen DOC App Pro Account.",
                        okButtonText: "OK"
                    }).then(function ( result ) {
                    });
                }, 500);
            }
        };
        DOCDelegate.ObjCProtocols = [UIApplicationDelegate];
        return DOCDelegate;
    })(UIResponder);
    application.ios.delegate = DOCDelegate;
}

**Comment by SaifulZiko** [comment]
After changing to the following code: 
**var errorIOS = new interop.Reference();
                          var content = NSString.alloc().initWithContentsOfURLEncodingError(url, NSUTF8StringEncoding, errorIOS);
                        console.log("content", content);
                        console.log("errorIOS", errorIOS);
                        console.log("errorIOS", errorIOS.value);**
Now App does not crash but File could not open or imported with NSapp 
got this error regarding permission : 
**CONSOLE LOG file:///app/bundle.js:487:24: applicationOpenURLOptions:  file:///private/var/mobile/Library/Mobile%20Documents/com~apple~CloudDocs/Downloads/file_Check.mcdc 
CONSOLE LOG file: app/app.js:305:0: content null 
CONSOLE LOG file: app/app.js:306:0: errorIOS <Reference: 0x2815cb2d0> 
CONSOLE LOG file: app/app.js:307:0: errorIOS Error Domain=NSCocoaErrorDomain Code=257 "The file \M-b\M^@\M^\file_Check.roe\M-b\M^@\M^] couldn\M-b\M^@\M^Yt be opened because you don\M-b\M^@\M^Yt have permission to view it." UserInfo={NSFilePath=/private/var/mobile/Library/Mobile Documents/com~apple~CloudDocs/Downloads/file_Check.mcdc, NSUnderlyingError=0x2819a5740 {Error Domain=N**

**Comment by janoshrubos** [comment]
Closing this issue due to inactivity. 
If this error is still relevant, feel free to open a new issue!
